index.html

```diff
@@ -0,0 +1,216 @@
+        @ Royal Victoria Dock 1 Western Gateway London E16 1XL London, UK
+      </div>
+    </section>
+
+    <!-- main program section -->
+    <section id="main-program" class="program-cards general-container">
+      <h2 class="main-program-heading">Main Program</h2>
+      <hr class="underline-mph" />
+      <div class="all-program-cards">
+        <div class="main-program-card">
+          <div class="image-for-program">
+            <img
+              src="./Images/icons/iron-man_75x75-removebg-preview.png"
+              alt=""
+              class="program-icon-logo"
+            />
+          </div>
+          <h3 class="program-title">Science & Tech</h3>
+          <p class="program-info">
+            It has become appallingly obvious that our technology has exceeded
+            our humanity.It has become appallingly obvious that our technology
+            has exceeded our humanity.
+          </p>
+        </div>
+        <div class="main-program-card">
+          <div class="image-for-program">
+            <img
+              src="./Images/icons/profesor-xabier_75x75-removebg-preview.png"
+              alt=""
+              class="program-icon-logo"
+            />
+          </div>
+          <h3 class="program-title">Eduction</h3>
+          <p class="program-info">
+            Man's mind, once stretched by a new idea, never regains its original
+            dimensions.
+          </p>
+        </div>
+        <div class="main-program-card">
+          <div class="image-for-program">
+            <img
+              src="./Images/icons/spiderman_75x75-removebg-preview.png"
+              alt=""
+              class="program-icon-logo"
+            />
+          </div>
+          <h3 class="program-title">Fun & Games</h3>
+          <p class="program-info">
+            All work an no play, makes Jack a dull boy.
+          </p>
+        </div>
+        <div class="main-program-card">
+          <div class="image-for-program">
+            <img
+              src="./Images/icons/thor-hammer_75x75.png"
+              alt=""
+              class="program-icon-logo"
+            />
+          </div>
+          <h3 class="program-title">Exercise & Health</h3>
+          <p class="program-info">
+            Good health and good sense are two of life's greatest blessings.
+          </p>
+        </div>
+        <div class="main-program-card">
+          <div class="image-for-program">
+            <img
+              src="./Images/icons/ant-man_1_75x75-removebg-preview.png"
+              alt=""
+              class="program-icon-logo"
+            />
+          </div>
+          <h3 class="program-title">Growth</h3>
+          <p class="program-info">
+            Be the change you want to see in this world.
+          </p>
+        </div>
+      </div>
+      <button class="join-program">Join Marvel Divide 2015</button>
+      <a href="" class="see-program">SEE THE WHOLE PROGRAME</a>
+    </section>
```

Comment:
- [ ] [REQUIRED] Good job so far! The `top` and `bottom` of the `<p/>` elements in the program section should be given the same amount of `padding` or `margin` as in [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), and don't forget to give your cards a `hover effect` as shown in [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)!

![image](https://user-images.githubusercontent.com/63915024/192009336-8b77766e-9491-497f-adef-d56105b15b17.png)

Reply:
thanks for the tip